articles/modules/ROOT/pages/realtime-querying-of-the-query-log-files.adoc

```diff
@@ -0,0 +1,85 @@
+= Realtime querying of the query.log files
```

Comment:
This is a really cool thing, but we should also point out that this approach may be a security risk, as other users may be able to search and query through the query logs this way, it doesn't look like it would be easy to restrict this approach to only certain users.